src/common/types/chainner-scope.ts

```diff
@@ -58,7 +58,7 @@ let PyTorchFaceModel = PyTorchModel {
 };
 let PyTorchSRModel = PyTorchModel {
     arch: invStrSet(PyTorchFaceModel.arch | PyTorchInpaintModel.arch),
-    subType: "SR"
+    subType: "SR" | "Restoration"
 };
 let PyTorchInpaintModel = PyTorchModel {
     arch: "LaMa" | "MAT",
```

Comment:
We should probably get rid of hard-coded arch names.

Reply:
Yes, absolutely.

Reply:
can we deal with that in a separate PR?

Reply:
Sure.